src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -239,4 +244,19 @@ public void startRegularSubmissionResultsPolling(@NotNull Project project) {
       regularSubmissionResultsPollingStarted = true;
     }
   }
+
+  /**
+   * Method that adds a new file (pattern) to the list of files not being shown in the Project UI.
+   *
+   * @param ignoredFileName a {@link String} name of the file to be ignored.
+   * @param project a {@link Project} to ignore the file from.
+   */
+  public static void ignoreFileInProjectView(@NotNull String ignoredFileName,
```

Comment:
unsure if it's the right place, but where it could then be? :confused:

Reply:
I guess `PluginSettings` is fine, we can always refactor after Sprint 3. If it's in `PluginSettings`, then perhaps make it non-static and call it as `PluginSettings.getInstance().ignore...`

Reply:
and will you then face the rage of Sonar instead of me? :rage:

Reply:
What does Sonar have to say about making this non-static? 🤔

Reply:
`Change this instance-reference to a static reference. ` but no it's gone... hm...

Reply:
made the change, the issue is gone now, forget it :smile: